model/job_queue/v1/queue_resource.model

```diff
@@ -27,10 +27,10 @@ resource Queue {
         out Kind String
         out HREF String
         in out Arguments String
-        out Attempts Integer
-        out AbandonedAt Date
+        in out Attempts Integer
+        in out AbandonedAt Date
         out ReceiptId String
-        out CreatedAt Date
+        in out CreatedAt Date
```

Comment:
So the user that pushes a new job can decide what is the creation date of that job?

Reply:
Talked yesterday to Brandon. Yes, you can re-push a job and to keep its creation time